boost-maven/boost-boms/boost-ee8-apis-bom/pom.xml

```diff
@@ -36,6 +36,16 @@
                 <artifactId>javax.transaction-api</artifactId>
                 <version>1.2</version>
             </dependency>
+            			<dependency>
+				<groupId>javax.enterprise</groupId>
+	            <artifactId>cdi-api</artifactId>
+				<version>2.0</version>
+			</dependency>
```

Comment:
Indentation above and below.

Reply:
it took two of us, but formatting should be right now